templates/quickstart-confluence-master.template

```diff
@@ -217,6 +217,11 @@ Parameters:
     - c4.2xlarge
     - c4.4xlarge
     - c4.8xlarge
+    - c5.large
```

Comment:
@avattathil Maybe I didn't follow all changes in this repo but why do we need 2 versions (onfluenceDataCenter.template and quickstart-confluence-master.template) of the template?